app/models/application_record.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
-  
+
   def self.search(search_params)
```

Comment:
Here it is! You can use this directly

Reply:
Even when you use a parent class' method, it's good to test it for this new use case (i.e. searching for pets)